pyproject.toml

```diff
@@ -13,6 +13,8 @@ license = "Apache 2.0"
 python = "^3.8"
 requests = "^2.25.1"
 singer-sdk = "0.42.1"
+sqlparse = "^0.5.3"
+pyhumps = "^3.8.0"
```

Comment:
Is this dependency actually used? If not, remove:

```console
$ poetry remove pyhumps
```

Reply:
It is: https://github.com/Matatika/tap-googleads/blob/1dc75e8f023681e62a056771831f24edad56583e/tap_googleads/dynamic_query_stream.py#L181C43-L181C51